CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## unreleased
 * Require Xcode 15.0+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
+* Security
+  * Updated expiring pinned vendor SSL certificates
```

Comment:
Typically we used the header as the name of the module, so using security here may be confusing. I think we can remove this header and have this at the same level as the Xcode 15 requirement which will indicate it applies to the entire SDK.
```suggestion
* Updated expiring pinned vendor SSL certificates
```

Reply:
Great thanks!